README.md

```diff
@@ -194,8 +194,8 @@ tokenManager: {
 | -------------- | ------------ |
 | `issuer`       | Specify a custom issuer to perform the OIDC flow. Defaults to the base url parameter if not provided. |
 | `clientId`     | Client Id pre-registered with Okta for the OIDC authentication flow. |
-| `redirectUri`  | The url that is redirected to when using `token.getWithRedirect`. This must be pre-registered as part of client registration. If no `redirectUri` is provided, defaults to the current origin. |
-| `postLogoutRedirectUri` | Specify the url where the browser should be redirected after [signOut](#signout). This url must be added to the list of `Logout redirect URIs` on the application's `General Settings` tab.
+| `redirectUri`  | The url that is redirected to when using `token.getWithRedirect`. This must be pre-registered as part of client registration. If no `redirectUri` is provided, defaults to the current origin (`window.location.origin`). |
+| `postLogoutRedirectUri` | Specify the url where the browser should be redirected after [signOut](#signout). This url must be added to the list of `Logout redirect URIs` on the application's `General Settings` tab. If not specified, your application's origin (`window.location.origin`) will be used.
```

Comment:
This change itself is fine, but looking at the other options, we might want to have more consistent language about defaults and highlighting which options are required.  (Compare clientId and pkce to the language on this option).

Reply:
> we might want to have more consistent language about defaults and highlighting which options are required. 

It's a good question. The docs say that only `url` is required, but that is going to be removed as part of this 3.0 process, and we will be requiring `issuer`. I assume `clientId` will also be required (but possibly not if using just as a node module and doing server-side auth). Needs further investigation.